drivers/wifi/nrf700x/zephyr/inc/zephyr_fmac_main.h

```diff
@@ -47,6 +47,10 @@ struct wifi_nrf_vif_ctx_zep {
 #ifdef CONFIG_WPA_SUPP
 	struct zep_wpa_supp_dev_callbk_fns supp_callbk_fns;
 #endif /* CONFIG_WPA_SUPP */
+	/* Used to store the negotaited twt flow id
+	 * for "twt_release_all" command.
+	 */
+	unsigned char neg_twt_flow_id;
```

Comment:
Just for future proofing, should we store make this an array?

Reply:
This is not part of the current release, only one flow id is being considered.

Reply:
Sure, that is why I mentioned future proofing, its simple to implement as well.